testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddLdapKeyStoreOfflineTest.java

```diff
@@ -0,0 +1,290 @@
+
+        OfflineManagementClient client = ManagementClient.offline(
+                OfflineOptions.standalone().configurationFile(cfg).build());
+
+        AddLdapKeyStore addLdapKeyStore = new AddLdapKeyStore.Builder("creaperKeyStore")
+                .dirContext("dirContext")
+                .searchPath("searchPath2")
+                .build();
+
+        assertXmlIdentical(SUBSYSTEM_SIMPLE, Files.toString(cfg, Charsets.UTF_8));
+        client.apply(addLdapKeyStore);
+
+        fail("Key store creaperKeyStore already exists in configuration, exception should be thrown");
+    }
+
+    @Test
+    public void overrideExisting() throws Exception {
+        File cfg = tmp.newFile("xmlTransform.xml");
+        Files.write(SUBSYSTEM_SIMPLE, cfg, Charsets.UTF_8);
+
+        OfflineManagementClient client = ManagementClient.offline(
+                OfflineOptions.standalone().configurationFile(cfg).build());
+
+        AddLdapKeyStore addLdapKeyStore = new AddLdapKeyStore.Builder("creaperKeyStore")
+                .dirContext("dirContext")
+                .searchPath("searchPath2")
+                .replaceExisting()
+                .build();
+
+        assertXmlIdentical(SUBSYSTEM_SIMPLE, Files.toString(cfg, Charsets.UTF_8));
+        client.apply(addLdapKeyStore);
+        assertXmlIdentical(SUBSYSTEM_EXPECTED_REPLACE, Files.toString(cfg, Charsets.UTF_8));
+    }
+
+    @Test
+    public void overrideNonExisting() throws Exception {
+        File cfg = tmp.newFile("xmlTransform.xml");
+        Files.write(SUBSYSTEM_SIMPLE, cfg, Charsets.UTF_8);
+
+        OfflineManagementClient client = ManagementClient.offline(
+                OfflineOptions.standalone().configurationFile(cfg).build());
+
+        AddLdapKeyStore addLdapKeyStore = new AddLdapKeyStore.Builder("creaperKeyStore2")
+                .dirContext("dirContext")
+                .searchPath("searchPath")
+                .replaceExisting()
+                .build();
+
+        assertXmlIdentical(SUBSYSTEM_SIMPLE, Files.toString(cfg, Charsets.UTF_8));
+        client.apply(addLdapKeyStore);
+        assertXmlIdentical(SUBSYSTEM_SECOND_KEY_STORE, Files.toString(cfg, Charsets.UTF_8));
+    }
+
+    @Test
+    public void addSecondKeyStore() throws Exception {
+        File cfg = tmp.newFile("xmlTransform.xml");
+        Files.write(SUBSYSTEM_SIMPLE, cfg, Charsets.UTF_8);
+
+        OfflineManagementClient client = ManagementClient.offline(
+                OfflineOptions.standalone().configurationFile(cfg).build());
+
+        AddLdapKeyStore addLdapKeyStore = new AddLdapKeyStore.Builder("creaperKeyStore2")
+                .dirContext("dirContext")
+                .searchPath("searchPath")
+                .build();
+
+        assertXmlIdentical(SUBSYSTEM_SIMPLE, Files.toString(cfg, Charsets.UTF_8));
+        client.apply(addLdapKeyStore);
+        assertXmlIdentical(SUBSYSTEM_SECOND_KEY_STORE, Files.toString(cfg, Charsets.UTF_8));
+    }
+
+//    @Test
```

Comment:
Is this test ingored intentionally? If so, could you please write a brief reason for that?